mmdet3d/datasets/convert_utils.py

```diff
@@ -459,14 +459,12 @@ def generate_waymo_mono3d_record(ann_rec, x1, y1, x2, y2, sample_data_token,
     repro_rec['bbox_corners'] = [x1, y1, x2, y2]
     repro_rec['filename'] = filename
 
-    coco_rec['file_name'] = filename
     coco_rec['image_id'] = sample_data_token
```

Comment:
please check whether we should keep `image_id`、`area` and `is_crowded` in `coco_rec`, whether they are used in `waymodataset`

Reply:
To my knowledge, these keys are not used now, but I'm not sure whether to use them in other conditions. @Tai-Wang

Reply:
these keys are used in v0.x and v1.0.0rcx version mono3d datasets, which inherits from coco dataset. Since we unify mono3d and lidar3d dataset, maybe it can be deprecated.